apps/src/applab/redux/screens.js

```diff
@@ -0,0 +1,27 @@
+const CHANGE_SCREEN = 'screens/CHANGE_SCREEN';
+
+const initialState = {
+  currentScreenId: null,
+  allScreens: []
```

Comment:
Yeah. This is an artifact of my optimisim :( I'll get rid of it until the point where we actually move ids into here.